bucket/ungoogled-chromium.json

```diff
@@ -1,35 +1,51 @@
+    "version": "79.0.3945.117",
+    "description": "Modifications to Google Chromium for removing Google integration and enhancing privacy, control, and transparency (macchrome's build)",
     "homepage": "https://github.com/Eloston/ungoogled-chromium",
-    "license": {
-        "identifier": "BSD-3-Clause",
-        "url": "https://github.com/Eloston/ungoogled-chromium/blob/master/LICENSE"
-    },
+    "license": "BSD-3-Clause",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/tangalbert919/ungoogled-chromium-binaries/releases/download/79.0.3945.117-1/ungoogled-chromium_79.0.3945.117-1.1_windows-x64.zip",
-            "hash": "64a20ae7bc8da0f987cb34dbd8430491cd5ec3082308498e479c8fd9db17e1bd",
-            "extract_dir": "ungoogled-chromium_79.0.3945.117-1.1_windows"
+            "url": "https://github.com/macchrome/winchrome/releases/download/v79.0.3945.117-r706915-Win64/ungoogled-chromium-79.0.3945.117-1_windows.7z",
+            "hash": "sha1:745c9dd30ec46cef517343bd953bb154ed44cbbb",
+            "extract_dir": "ungoogled-chromium-79.0.3945.117-1_windows"
+        },
+        "32bit": {
+            "url": "https://github.com/macchrome/winchrome/releases/download/v79.0.3945.117-r706915-Win64/Ungoogled-Chromium-79.0.3945.117-Win32.7z",
+            "hash": "sha1:907094d8a3dd925b6db490bd8d65d50a3419f866",
+            "extract_dir": "Ungoogled-Chromium-79.0.3945.117-Win32"
         }
     },
-    "bin": "chrome.exe",
+    "bin": [
+        "chrome.exe",
+        [
+            "chrome.exe",
+            "chromium"
+        ]
+    ],
     "shortcuts": [
         [
             "chrome.exe",
             "Ungoogled Chromium"
         ]
     ],
     "checkver": {
-        "url": "https://ungoogled-software.github.io/ungoogled-chromium-binaries/releases/windows/64bit/",
-        "re": "\\d\">\\s*([\\d.-]+)<"
+        "github": "https://github.com/macchrome/winchrome",
+        "regex": "releases/tag/v([\\d.]+)-r(?<rev>\\d+)-Win64"
     },
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://github.com/tangalbert919/ungoogled-chromium-binaries/releases/download/$version/ungoogled-chromium_$version.1_windows-x64.zip",
-                "extract_dir": "ungoogled-chromium_$version.1_windows"
+                "url": "https://github.com/macchrome/winchrome/releases/download/v$version-r$matchRev-Win64/ungoogled-chromium-$version-1_windows.7z",
+                "extract_dir": "ungoogled-chromium-$version-1_windows"
+            },
+            "32bit": {
+                "url": "https://github.com/macchrome/winchrome/releases/download/v$version-r$matchRev-Win64/Ungoogled-Chromium-$version-Win32.7z",
+                "extract_dir": "Ungoogled-Chromium-$version-Win32"
             }
+        },
+        "hash": {
+            "url": "https://github.com/macchrome/winchrome/releases/tag/v$version-r$matchRev-Win64",
+            "regex": "(?s)$basename.*?$sha1"
```

Comment:
Because the merge of 32-bit version and 64-bit version, the hash autoupdate will fail for 32-bit version.